app/controllers/web/admin/domains.js

```diff
@@ -44,6 +45,27 @@ async function list(ctx) {
   ctx.body = { table };
 }
 
+async function remove(ctx) {
+  const domain = await Domains.findById(ctx.params.id);
+
+  if (!domain) throw Boom.notFound(ctx.translateError('INVALID_USER'));
```

Comment:
You're using the wrong phrase here, it should be `'INVALID_DOMAIN'`

Reply:
oh i thought i caught these. also it isn't [`INVALID_DOMAIN`](https://github.com/forwardemail/forwardemail.net/blob/feat/admin-domains/config/phrases.js#L55) it is  [`DOMAIN_DOES_NOT_EXIST`](https://github.com/forwardemail/forwardemail.net/blob/05b44b2ddb3a4c2e74ff2479f1b5d487a9c42118/config/phrases.js#L62)

Reply:
this is resolved in another commit.